suzieq/cli/sqcmds/context_commands.py

```diff
@@ -28,15 +28,32 @@
     "datadir",
     description="Set the data directory for the command"
 )
+@argument(
+    'rest_server_ip', description='IP address of the REST server'
+)
+@argument(
+    'rest_server_port', description='Port of the REST server'
+)
+@argument(
+    'rest_api_key', description='API key for the REST server'
+)
+@argument(
+    'rest_use_https', description='Use HTTPS for the REST server',
+    choices=['True', 'False']
+)
 def set_ctxt(
```

Comment:
We need to change the description for the engine argument

Reply:
Ack